opensearch-jobs.yml

```
@@ -162,7 +162,7 @@ instance_groups:
           daily: true
           hourly: false
         purge_logs:
-          retention_period: 90
+          retention_period: 180
```

```suggestion
          retention_period: 365
```

By default, we're going to keep logs in Opensearch for 365 days (1 year) to meet our compliance obligations. Then we can override for lower environments like dev and staging to save money, as you have

---

Perfect